BUILD.md

```diff
@@ -0,0 +1,16 @@
+## Build locally
```

Comment:
This seems to be a duplication of what's in the README. Maybe worth removing it from there and linking to this doc.

Related: What's the reason for the additional file?

Reply:
I updated the BUILD.md to be a little more NOOB friendly. I didn't modify README.md but I think those instructions in README.md should be removed at some point.